### ISSUE TITLE
Any support for macOS?

### ISSUE THREAD

**Issue by ghost** [alexst07/shell-plus-plus#4]
# MacOS support

    Do you want to add macOS support?

**Comment by alexst07** [comment]
Yes, I want, I just haven't tried to compile for macOS because I don't have the OS yet, but I will try to provide one to try compiling for it. I believe that by design using technologies that also support macOS, it should compile without major problems.

**Comment by ghost** [comment]
I have the macOS, I can help you

**Comment by ghost** [comment]
And can I compile Shell++ in Windows 10?

**Comment by alexst07** [comment]
> I have the macOS, I can help you

Your help is very welcome

> And can I compile Shell++ in Windows 10?

At the moment not, the part responsible for the execution of the processes uses Posix, what I intend to do is to use the Boost process, so it will be possible to compile also in Windows.

**Comment by ghost** [comment]
Sorry for trouble... How to read user input in Shell++ like in python input() or bash read to the variable?

**Comment by alexst07** [comment]
```
a = read()
print(a)
```

**Comment by alexst07** [comment]
@entyav, in fact I apologize, so far this has been a personal project and for fun. However, whatever you need you can ask, and if you are interested in the project, I can go on making documentation with your doubts, as it is not yet used in production, we can make changes if you have any suggestions.
